website/files/exceptions.py

```diff
@@ -8,3 +8,6 @@ class SubclassNotFound(FileException):
 
 class VersionNotFoundError(FileException):
     pass
+
+class FileNodeorChildCheckedOutError(FileException):
```

Comment:
Maybe rename `FileNodeCheckedOutError`? You can explain in the docstring that this will be raised if a filenode or its children is checked out.